lldb/tools/lldb-dap/package.json

```diff
@@ -348,6 +348,17 @@
                 "type": "string",
                 "description": "The time in seconds to wait for a program to stop when attaching using \"attachCommands\". Defaults to 30 seconds."
               },
+              "port": {
+                "type": [
+                  "number",
+                  "string"
+                ],
+                "description": "TCP/IP port to attach to. Specifying both pid and port is an error."
+              },
```

Comment:
Do we want to label this `"gdb-remote-port"` instead of `"port"`? I don't think we have any other process plug-ins that support connecting via a port, so it migth be nice to let people know what this is for?

Reply:
Thank you for the suggestion. I think labeling it as "gdb-remote-port" & "gdb-remote-hostname" makes sense for clarity. Since there are no other process plugins that support connecting via port, this lable will also help users understand its purpose more easily.
We used the attribute names "port" and "hostname" because Microsoft visual studio Code IDE examples use the same terminology (https://code.visualstudio.com/Docs/editor/debugging#_launch-configurations). Please reconsider this suggestion to ensure consistency with widely accepted convention.


![image](https://github.com/llvm/llvm-project/assets/150406203/409b9b42-64fc-46c4-a152-94b3357f0488)

Reply:
I think we should make it clear by prefixing with `"gdb-remote-port"` and `"gdb-remote-hostname"` to make it clear this is setting up the GDB remote debug configuration. It then also means we can hard code the `"gdb-remote"` plug-in name as these settings imply that process plugin. The word `"port"` is too general and we might want to use for connecting to a `lldb-dap` process that is running and listening on another host/port as well.

Reply:
Renamed "port" and "hostname" to "gdb-remote-port" and "gdb-remote-hostname"